stdlib/planner/group_agg_test.flux

```diff
@@ -1,338 +1,134 @@
-        |> range(start: -100y)
-        |> filter(fn: (r) => r._field == "f0")
+    want = array.from(
+        rows: [
+            {"t0": "t0v0", "_value": 6},
+            {"t0": "t0v1", "_value": 6},
+        ],
+    )
         |> group(columns: ["t0"])
-        |> count()
-        |> drop(columns: ["_start", "_stop"])
-    testing.diff(got, want)
-}
-
-testcase group_one_tag_and_field_filter_field_count {
-    want = array.from(rows: [
-        {"t0": "t0v0", "_field": "f0", "_value": 6},
-        {"t0": "t0v1", "_field": "f0", "_value": 6},
-    ]) |> group(columns: ["t0", "_field"])
-    got = testing.loadStorage(csv: input)
+    got = testing.load(tables: inData)
         |> range(start: -100y)
         |> filter(fn: (r) => r._field == "f0")
-        |> group(columns: ["t0", "_field"])
-        |> count()
-        |> drop(columns: ["_start", "_stop"])
-    testing.diff(got, want)
-}
-
-testcase group_all_tags_filter_field_count {
-    want = array.from(rows: [
-        {"t0": "t0v0", "t1": "t1v0", "_value": 3},
-        {"t0": "t0v1", "t1": "t1v0", "_value": 3},
-        {"t0": "t0v0", "t1": "t1v1", "_value": 3},
-        {"t0": "t0v1", "t1": "t1v1", "_value": 3},
-    ]) |> group(columns: ["t0", "t1"])
-    got = testing.loadStorage(csv: input)
-        |> range(start: -100y)
-        |> filter(fn: (r) => r._field == "f0")
-        |> group(columns: ["t0", "t1"])
+        |> group(columns: ["t0"])
         |> count()
         |> drop(columns: ["_start", "_stop"])
-    testing.diff(got, want)
-}
 
-testcase group_all_tags_and_field_filter_field_count {
-    want = array.from(rows: [
-        {"_field": "f0", "t0": "t0v0", "t1": "t1v0", "_value": 3},
-        {"_field": "f0", "t0": "t0v1", "t1": "t1v0", "_value": 3},
-        {"_field": "f0", "t0": "t0v0", "t1": "t1v1", "_value": 3},
-        {"_field": "f0", "t0": "t0v1", "t1": "t1v1", "_value": 3},
-    ]) |> group(columns: ["t0", "t1", "_field"])
-    got = testing.loadStorage(csv: input)
-        |> range(start: -100y)
-        |> filter(fn: (r) => r._field == "f0")
-        |> group(columns: ["t0", "t1", "_field"])
-        |> count()
-        |> drop(columns: ["_start", "_stop"])
-    testing.diff(got, want)
+    testing.diff(got, want) |> yield()
 }
 
 // Group + sum tests
-
-// Group + sum tests with no filter on field
-
-testcase group_all_sum {
-    want = array.from(rows: [{"_value": 41}])
-    got = testing.loadStorage(csv: input)
-        |> range(start: -100y)
-        |> group()
-        |> sum()
-        |> drop(columns: ["_start", "_stop"])
-    testing.diff(got, want)
-}
-
 testcase group_one_tag_sum {
-    want = array.from(rows: [
-        {"t0": "t0v0", "_value": 19},
-        {"t0": "t0v1", "_value": 22},
-    ]) |> group(columns: ["t0"])
-    got = testing.loadStorage(csv: input)
-        |> range(start: -100y)
+    want = array.from(
+        rows: [
+            {"t0": "t0v0", "_value": 19},
+            {"t0": "t0v1", "_value": 22},
+        ],
+    )
```

Comment:
This test case is definitely simpler to read. Do we not need a test case with multiple tags though?